client/packages/host/src/api/hooks/utils/useHostApi.ts

```diff
@@ -5,7 +5,10 @@ import { getSdk } from '../../operations.generated';
 export const useHostApi = () => {
   const keys = {
     base: () => ['host'] as const,
-    settings: () => [...keys.base(), 'settings'] as const,
+    syncSettings: () => [...keys.base(), 'syncSettings'] as const,
+    syncState: () => [...keys.base(), 'syncState'] as const,
+    syncStatus: () => [...keys.base(), 'syncStatus'] as const,
+    initialiseSite: () => [...keys.base(), 'initialiseSite'] as const,
```

Comment:
I tried to follow existing pattern and just added a more specific key since more mutations are added, previously :

https://github.com/openmsupply/open-msupply/blob/ebf8cf93d3c89f48d428e039c32adfff4a2a7887/client/packages/host/src/api/hooks/sync/useSyncSettingsUpdate.ts#L19

I would really like to have an example repo/module that shows what are all the different options of queryClient mutations/queries that we use and how each key/option affects it, we might have documentation that I've missed ?